device-types/Ciena/waveserver-5.yaml

```diff
@@ -0,0 +1,98 @@
+---
+manufacturer: Ciena
+model: Waveserver 5
+slug: ciena-waveserver-5
+part_number: 186-3001-900
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it